frontend/src/queries/nodes/InsightViz/InsightViz.scss

```diff
@@ -0,0 +1,109 @@
+.InsightViz {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    gap: 1rem;
+    container-type: inline-size;
+    flex: 1; // important for when rendered within a flex parent
+
+    &.InsightViz--horizontal {
+        flex-direction: row;
+        flex-wrap: wrap;
+
+        .EditorFiltersWrapper {
+            width: 100%;
+
+            @container (min-width: 768px) {
+                width: 30%;
+                min-width: 26rem;
+                max-width: 30rem;
+            }
+        }
+    }
+}
+
+.InsightVizDisplay {
+    --insight-viz-min-height: calc(80vh - 4rem);
+    display: flex;
+    flex-direction: column;
+    min-height: var(--insight-viz-min-height);
+
+    .InsightVizDisplay__content {
+        display: flex;
+        flex: 1;
+        position: relative;
+
+        .InsightVizDisplay__content__left {
+            display: flex;
+            flex-direction: column;
+            flex: 1;
+            position: relative;
+            width: 100%;
+        }
+
+        .InsightVizDisplay__content__right {
+            flex-shrink: 1;
+            max-width: 45%;
+            height: var(--insight-viz-min-height);
+            width: fit-content;
+            margin: 1rem;
+            display: flex;
+            align-items: center;
+        }
+    }
+
+    .InsightDisplayConfig {
+        border-bottom-width: 1px;
+        padding: 0.5rem;
+    }
+}
+
+.TrendsInsight {
+    // &--ActionsLineGraph
```

Comment:
Here for reference what the specific values are - from now on if we need custom styling here is the place to do it